src/utils/Scale.js

```diff
@@ -147,10 +131,21 @@ export function invertPointScale(scale, rangeValue) {
   // shim until d3.scalePoint.invert() is implemented for real
   // given a value from the output range, returns the *nearest* corresponding value in the input domain
   const rangePoints = domain.map(domainValue => scale(domainValue));
-  const nearestLeftPointIndex = indexOfClosestLeftNumberInList(
-    rangeValue,
-    rangePoints
-  );
 
-  return domain[nearestLeftPointIndex];
+  if (rangePoints.length <= 1) {
```

Comment:
is there a use case where there is a 0? is the undefined caught in this use case?

Reply:
good point! will add in next release

Reply:
inspecting d3's code domain looks like it'll always return an array https://github.com/d3/d3-scale/blob/master/src/continuous.js#L91